src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -93,224 +93,17 @@ struct ComputeDuDt {
 };
 
 /*!
- * \brief Compute normal component of flux on a boundary.
- *
- * \f{align}
- * F(\Psi) =& 0 \\
- * F(\Pi) =& n^i \Phi_i \\
- * F(\Phi_i) =& n_i \Pi
- * \f}
+ * \brief A relic of an old incorrect way of handling boundaries for
```

Comment:
Is this going to be removed?

Reply:
Yes, as part of reworking the way the time derivatives and boundary conditions are handled I'm going to add a way for the boundary correction terms to list which evolved variables actually need an `n_i F^i` subtracted off. That'll add support for the weak form and also make it easy to remove these functions.